core/src/main/java/bio/terra/pearl/core/service/datarepo/DatasetService.java

```diff
@@ -26,21 +26,17 @@ public DatasetService(DatasetDao dao, DataRepoClient dataRepoClient) {
     public void delete(Dataset dataset) {
         dao.delete(dataset.getId());
         try {
-            JobModel jobModel = dataRepoClient.deleteDataset(dataset.getDatasetId());
-            logger.info("Deleted dataset: {} (TDR job ID {})", dataset.getDatasetId(), jobModel.getId());
+            JobModel jobModel = dataRepoClient.deleteDataset(dataset.getTdrDatasetId());
+            logger.info("Deleted dataset: {} (TDR job ID {})", dataset.getId(), jobModel.getId());
         } catch (ApiException e) {
-            logger.error("Unable to delete dataset {}. Error: {}", dataset.getDatasetId(), e.getMessage());
+            logger.error("Unable to delete dataset {}. Error: {}", dataset.getId(), e.getMessage());
```

Comment:
These used to all use `dataset.getDatasetId()`. I'm guessing that you intentionally chose between `dataset.getTdrDatasetId()` and `dataset.getId()` here... and this is probably exactly the kind of thing you're clearing up with this refactor!